docs/release_docs/AHA-Prod_ReleaseFlow.md

```diff
@@ -0,0 +1,36 @@
+# AHA-Prod Workflow
+Explaination of AHA-PROD workflow.
+![AHA-PROD](https://github.com/REAN-Foundation/reancare-service/blob/feature/flow_documentation/assets/images/AHA-PROD_Workflow.png?raw=true)
+
+## AHA-PROD-ci-cd
+
+This workflow allows you to manually trigger a GitHub Action with a input Github ``` Tag_name ```, without having to push or create a pull request
+
+### JOBS
+
+#### Github-ECR-Tag-Check
+This is how Github-ECR-Tag-Check job looks like
```

Comment:
Instead of this please write:
The GitHub (G & H are always uppercase in GitHub) ECR Tag Check will be performing the following steps. 

(We don't need to show how the job code looks like, instead we can show what all will be the steps executed by the job)